jepsen 0.2.7-sn  AgentProxyException: connector is not available

If I run test using  [jepsen "0.2.7-SNAPSHOT"]

i got
```
Caused by: com.jcraft.jsch.agentproxy.AgentProxyException: connector is not available:
        at com.jcraft.jsch.agentproxy.ConnectorFactory.createConnector(ConnectorFactory.java:120)
        at jepsen.control.sshj$agent_proxy.invokeStatic(sshj.clj:38)
        at jepsen.control.sshj$agent_proxy.invoke(sshj.clj:36)
        at jepsen.control.sshj$auth_BANG_$fn__6739.invoke(sshj.clj:54)
        at jepsen.control.sshj$auth_BANG_.invokeStatic(sshj.clj:53)
        at jepsen.control.sshj$auth_BANG_.invoke(sshj.clj:42)
        at jepsen.control.sshj.SSHJRemote.connect(sshj.clj:118)
```

When using   [jepsen "0.2.1-SNAPSHOT"]

```
         "jepsen.checker$unbridled_optimism$reify__8609@470c4229"]
 :argv ("test" "-n" "n1" "n2" "n3")
 :nemesis
 #object[jepsen.nemesis$reify__9186
         "0x6408be5"
         "jepsen.nemesis$reify__9186@6408be5"]
 :active-histories
 #object[clojure.lang.Atom "0x549d14fe" {:status :ready, :val #{}}]
 :nodes ["n1"]
 :test-count 1
 :generator nil
 :os
 #object[jepsen.os.centos.CentOS
         "0x2e44cb34"
         "jepsen.os.centos.CentOS@2e44cb34"]
 :time-limit 60}

INFO [2022-08-10 12:02:28,155] jepsen node n1 - jepsen.os.centos n1 setting up centos
INFO [2022-08-10 12:02:30,013] jepsen node n1 - jepsen.util #{"iproute"
  "curl"
  "gcc-c++"
  "logrotate"
  "gcc"
  "rsyslog"
  "unzip"
  "vim-common"
  "wget"
  "ncurses-devel"}
```

it works ok...

any clues?

--- a/jepsen/src/jepsen/control/sshj.clj
+++ b/jepsen/src/jepsen/control/sshj.clj
@@ -10,6 +10,7 @@
                             [scp :as scp]]
             [slingshot.slingshot :refer [try+ throw+]])
   (:import (com.jcraft.jsch.agentproxy AgentProxy
+                                       AgentProxyException
                                        ConnectorFactory)
            (com.jcraft.jsch.agentproxy.sshj AuthAgent)
            (net.schmizz.sshj SSHClient)
@@ -59,6 +60,8 @@
               methods (auth-methods agent-proxy)]
           (.auth c ^String username methods)
           true)
+        (catch AgentProxyException e
+          false)
         (catch UserAuthException e
           false))
 

